AutWebTest/AutWebTest/Helpers/Helper.cs

```diff
@@ -0,0 +1,17 @@
+﻿using AutWebTest.Pages;
+using FizzWare.NBuilder;
+using OpenQA.Selenium;
+using OpenQA.Selenium.Chrome;
+
+namespace AutWebTest.Helpers
+{
+    public static class Helper
+    {
+        public static string RandomString(int length)
```

Comment:
Methods should have some verb in title, e.g. GetRandomString, GenerateRandomString, etc.

```suggestion
        public static string GetRandomString(int length)
```